packages/astro/src/core/dev/restart.ts

```diff
@@ -30,7 +30,7 @@ async function createRestartedContainer(
 	return newContainer;
 }
 
-const configRE = /.*astro.config.(?:mjs|cjs|js|ts)$/;
+const configRE = /.*astro.config.(?:mjs|mts|cjs|js|ts)$/;
```

Comment:
It looks like we're missing `cts` here too, can we update it here too?
```suggestion
const configRE = /.*astro.config.(?:mjs|mts|cjs|cts|js|ts)$/;
```

https://github.com/withastro/astro/blob/676b2c66ee5d991bfb8ea85ca466be690cf0e5b9/packages/astro/src/core/config/config.ts#L29-L36